extract/raw_array_cohort_extract.py

```diff
@@ -166,12 +182,14 @@ def do_extract(fq_dataset,
   parser.add_argument('--dataset',type=str, help='project.dataset location of raw array data', required=True)
   parser.add_argument('--fq_destination_table',type=str, help='fully qualified destination table', required=True)
   parser.add_argument('--query_project',type=str, help='Google project where query should be executed', required=True)
-  parser.add_argument('--fq_cohort_sample_mapping_table',type=str, help='Mapping table from sample_id to sample_name for the extracted cohort', required=True)
+  parser.add_argument('--sample_mapping_table',type=str, help='Mapping table from sample_id to sample_name', required=True)
```

Comment:
Could you name this `fq_sample_mapping_table` since it still needs to be fully qualified? Or add that to the help comment?